src/client/src/components/icons/icons.stories.tsx

```diff
@@ -1,10 +1,10 @@
-import { storiesOf} from "@storybook/react"
+import { storiesOf } from "@storybook/react"
 import * as icons from "./"
 
 const stories = storiesOf("Components/Icons", module)
 
 Object.entries(icons).forEach(([key, Icon]: any) =>
   stories.add(key, () => {
-    return typeof Icon === "function" ? <Icon /> : Icon
+    return typeof Icon === "function" ? <Icon width={24} height={24} /> : Icon
```

Comment:
width/height 24 is not appropriate for all icons .. did you test the whole set?

suggest:
```
    return Icon.type && Icon.type === "svg" ? Icon : <Icon />
```

so relying on defaults, which is good to test what the default looks like.
In some cases, defaults are just not used, though .. see other comment on `LogoIcon`
Still good practice to use a 'sensible' default ... sometimes _sensible_ is the opinion of UX